src/features/search/components/__tests__/SearchResults.native.test.tsx

```diff
@@ -91,6 +91,12 @@ describe('SearchResults component', () => {
         type: 'SET_STATE',
         payload: mockSearchState,
       })
+    })
+
+    it('should redirect on filters page on location button click', async () => {
+      const { getByTestId } = render(<SearchResults />)
+      const locationButton = getByTestId('locationButton')
+      await fireEvent.press(locationButton)
```

Comment:
je propose un p'tit saut de ligne pour séparer les 3 parties du test

```suggestion
      const { getByTestId } = render(<SearchResults />)
      const locationButton = getByTestId('locationButton')

      await fireEvent.press(locationButton)
```